elements/pfelement/src/pfelement.js

```diff
@@ -885,23 +890,37 @@ class PFElement extends HTMLElement {
     return propName;
   }
 
-  _copyAttributes(selectors, set) {
+  _cascadeAttributes(selectors, set) {
     selectors.forEach(selector => {
       set[selector].forEach(attr => {
-        this._copyAttribute(attr, selector);
+        this._cascadeAttribute(attr, selector);
       });
     });
   }
 
-  _copyAttribute(name, to) {
+  /**
+   * Trigger a cascade of the named attribute to any child elements that match
+   * the `to` selector.  The selector can match elements in the light DOM and
+   * shadow DOM.
```

Comment:
Can you add the `@param` and `@returns` to this too?

Reply:
Done!  I skipped `@returns` because this function returns nothing (therefore implicit return of `undefined`), which JSDoc understands and omits the return type.